io/plugins/it.geosolutions.hale.io.appschema/src/it/geosolutions/hale/io/appschema/writer/internal/AbstractPropertyTransformationHandler.java

```diff
@@ -607,7 +637,7 @@ private void setEncodeIfEmpty(ClientProperty clientProperty) {
 				attributeMapping.setEncodeIfEmpty(true);
 			}
 		} catch (CQLException e) {
-			log.warn("Cannot set encodeIfEmpty value. Reason: " + e.getMessage());
+			LOG.warn("Cannot set encodeIfEmpty value. Reason: " + e.getMessage());
```

Comment:
The message should say that the exception \ situation is ignore and skipped, unless that's not the intention.